README.md

```diff
@@ -56,10 +52,18 @@ property in the `uses` block for the action:
     working-directory: src
 ```
 
-| variable          | default | what it does                                                                                                    |
-|-------------------|---------|-----------------------------------------------------------------------------------------------------------------|
-| lint-glob         | `.`     | Indicates what files to run eslint on. This is any single glob that eslint supports.                            |
-| working-directory | `.`     | Indicates where install [@18f/18f-eslint](https://npm.im/@18f/18f-eslint). This path must include package.json. |
+| variable          | default | what it does                                                                                                       |
+|-------------------|---------|--------------------------------------------------------------------------------------------------------------------|
+| lint-glob         | `.`     | Indicates what files to run eslint on. This is any single glob that eslint supports.                               |
+| only-changed      | `false` | Indicates whether to only run eslint on changed files. Must be `true` to enable. This will override `lint-glob`.   |
+| working-directory | `.`     | Indicates where to install [@18f/18f-eslint](https://npm.im/@18f/18f-eslint). This path must include package.json. |
```

Comment:
I think `This path must include package.json.` could be a little clearer. Maybe `your package.json must be in this directory` ?

I can't quite tell wether it's valid for package.json to be in `working-directory/foo`, for instance, or if I should include `package.json` in the value when I provide it